JavaScript/Merging_PDF/README.md

```diff
@@ -0,0 +1,6 @@
+# Merging PDF
```

Comment:
The PDF doesn't align with the template we use 

[Here](https://github.com/HarshCasper/Rotten-Scripts/blob/master/TEMPLATE_README.md) is the template. Please update accordingly.